packages/core-components/src/lib/unsorted/ui/Tabs/Tabs.svelte

```diff
@@ -16,23 +16,27 @@
 	 * color can be provided to set custom background color for active tab.
 	 * @type {string}
 	 */
-	export let color = '#268aed'; // or any 6 digit hex color code.
-	const colorArray = hexToRgbArray(color);
+	export let color = '#268aed';
+	color = color.replace(/\s+/g, '').toLowerCase(); // clean string
 
-	function hexToRgbArray(hexColor) {
-		const hexDigits = hexColor.replace(/\s+/g, '').replace('#', '');
+	const bgColor = isValidColorString(color) ? addOpacityToColor(color) : '#268aed1a';
+	const borderColor = isValidColorString(color) ? color : '#268aed';
 
-		// Check if the provided color contains valid hex characters.
-		if (!/^[0-9a-fA-F]{6}$/.test(hexDigits)) {
-			return null;
-		}
+	function isValidColorString(inputColor) {
+		const hexRegex = /^#([0-9A-Fa-f]{3}|[0-9A-Fa-f]{6})$/;
```

Comment:
Since you've done the hard work to create the regex strings, should you put them into an e.g. isHex(inputColor) function which you can then reuse below when adding opacity?